staubli_tx2-90_support/package.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<package>
```

Comment:
Could you please convert this to format 2?

See [staubli_tx60_support/package.xml](https://github.com/ros-industrial/staubli_experimental/blob/b4a0d23e54c7f48b5f043314517d782e1cc344de/staubli_tx60_support/package.xml) for an example.